specs/darwin/temps.table

```diff
@@ -0,0 +1,9 @@
+table_name("temps")
+description("Temperature sensors.")
+schema([
+    Column("smc_key", TEXT, "SMC key", additional=True),
```

Comment:
Sorry about not seeing this earlier, but `additional=True` should only be included if you're using the `QueryContext` data within the table. It's silly of me to not document this better but the additional means you may return different results based on constraints in there where clause: http://osquery.readthedocs.org/en/stable/development/creating-tables/

Since there's no constraints optimization implementation in this table, just remove this keyword argument.